hackathon_site/hardware/serializers.py

```diff
@@ -198,6 +200,9 @@ def validate(self, data):
             user_profile = self.context["request"].user.profile
         except ObjectDoesNotExist:
             raise serializers.ValidationError("User does not have profile")
+        team_size = Profile.objects.filter(team__exact=user_profile.team).count()
+        if team_size < MIN_MEMBERS or team_size > MAX_MEMBERS:
+            raise serializers.ValidationError("Does not match team size criteria")
```

Comment:
normally, we relate error messages to the user (because the user will be submitting the order, not the team), could you change this message to `"User's team does not meet team size criteria"`